src/CookiePhysicsComponent.h

```diff
@@ -3,6 +3,13 @@
 
 #include "PhysicsComponent.h"
 
+typedef struct {
+    int hits;
+    double launchTime;
+    glm::vec3 launchPosition;
+    glm::vec3 hitPosition;
```

Comment:
We can make it a lot of things. You can get quite philosophical how the scoring should work. We also could just use the distance from launch to the first hit to calculate the distance multiplier, and then use this multiplier for all hits it scores.